oranda.json

```diff
@@ -13,7 +13,11 @@
     "twitter_account": "@axodotdev"
   },
   "artifacts": {
-    "cargo_dist": true
+    "cargo_dist": true,
+    "package_managers": {
+      "npm": "npm install oranda",
```

Comment:
do we want this to take a full string or can we construct this from the name of the pkg manager alone?

Reply:
I would rather take the string otherwise every time someone makes a new package manager we need to write crappy code

Reply:
maybe this can be a thing we have cargo dist do ;)